test/ClangImporter/cxx_interop_sil.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: %target-swift-frontend -module-name cxx_sil -I %S/Inputs/custom-modules -module-cache-path %t -enable-cxx-interop -emit-sil -o - -primary-file %s | %FileCheck %s
+
+import CXXInterop
+
+// CHECK-LABEL: sil hidden @$s7cxx_sil13virtualMethod1as5Int32VSpySo14MethodsVirtualVG_tF
+// CHECK: [[INT_LIT:%.*]] = integer_literal $Builtin.Int32, 7
+// CHECK: [[INT:%.*]] = struct $Int32 ([[INT_LIT]] : $Builtin.Int32)
+// CHECK: [[THIS_PTR:%.*]] = begin_access [modify] [unsafe] {{%.*}} : $*MethodsVirtual
+// CHECK: ([[VIRTUAL_METHOD:%.*]], [[THIS_PTR_ADJUST:%.*]]) = cxx_virtual_method [[THIS_PTR]] : $*MethodsVirtual, #MethodsVirtual.SimpleVirtualMethod!1.foreign : (inout MethodsVirtual) -> (Int32) -> Int32, $@convention(c) (@inout MethodsVirtual, Int32) -> Int32
+// TODO: Use adjusted this ptr here...
```

Comment:
Even without a truly adjusted this-ptr, you could still test with a single-inheritance virtual dispatch.

Reply:
I'm not sure what you mean. I will note that if you have:
```c++
class A { virtual void test() = 0; };
class B : public A {};
```
In swift, test() will only be accessible on a pointer of type A. I think any other test will be essentially equivalent to what I have here, but I'd be happy to add another test if you come up with one.

Reply:
Oh, right, because we're not importing these as classes. Okay, that'll be Fun.